export const work_experience = [
  {
    id: 1,
    institution: 'Tempus Labs',
    startDate: 'March 2019',
    endDate: 'Present',
    title: 'Senior Clinical Scientist',
    duties: [
      {
        duty:'Created and maintained cross-instituional SOPs for variant classification'
      },
      {
        duty: 'Identified and implemented new worlflows to harmonize variant classification across multiple institutions'
      },
      {
        duty: 'Designed and implemented reportable bed file to report only specific genomic regions for Tempus xT and xF assays'
      },
      {
        duty: 'Used R programming language to design and implement a regions of interest bed file for reporting insertions and deletions for Tempus xF assay'
      },
      {
        duty: 'Used R programming language to create, update, and modify curated data to justify the inclusion or exclusion of genomic regions for xT-oncology FDA application'
      },
      {
        duty: 'Acted as the scientific lead in the development of the Tempus xF liquid biopsy panel to sequence and report SNVs and indels in clinically relevant regions of 105 genes plus CNVs and DNA rearrangements in a subset of these genes'
      },
      {
        duty: 'Used R programming language to programmatically access data, wrangle data, analyze data and present data to implement changes in workflow or other internal processes'
      }
    ]
  },
  {
    id: 2,
    institution: 'Tempus Labs',
    startDate: 'June 2017',
    endDate: 'Febuary 2019',
    title: 'Clinical Scientist',
    duties: [
      {
        duty: 'Evaluated data and analyzed variants for a project that lead to the publication of the results in Nature BioTechnology'
      },
      {
        duty: 'Trained over 20 M.A.s, Ph.D.’s and post-baccalaureate in germline and somatic variant classification'
      },
      {
        duty: 'Worked closely with colleagues to build a team of variant scientists from 2 individuals to over 20 highly skilled variant scientists'
      },
      {
        duty: 'ained proficiency in the R programming language and experience using Python, JavaScript, HTML, and CSS. '
      }
    ]
  },
  {
    id:3,
    institution: `Harvard Medical School, Partners Healthcare, Brigham and Women's Hospital`,
    startDate: 'June 2015',
    endDate: 'June 2017',
    title: 'Clinical Knowlege Curation Coordinator (Variant Classification Scientist)',
    duties: [
        {
            duty: 'Trained over 25 M.Ds, M.D./Ph.D.’s, rotating fellows, undergraduates, and post-baccalaureate in constitutional variant classification'
        },
        {
            duty:'Assessed pathogenicity of variants from large gene panels covering cardiomyopathy, hearing loss, pulmonary disease, and Noonan syndrome'
        },
        {
            duty:'Evaluated data and assessed variants as a critical member of the Geisinger MyCode project including the 56 genes recommended by the ACMG and 20 additional genes from Geisinger'
        },
        {duty:'Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)'},
        {
            duty:`Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)
Proficient in use of variant databases including- HGMD, ClinVar, LOVD, Deafness Variation database, Cardiodb, MitoMap, Leiden Muscular Dystrophy database, ARUP and COSMIC`
        }
    ]
  },
  {
    id:4,
    institution:`EBSCO Information Services, DynaMed`,
    startDate:'January 2015',
    endDate:'June 2015',
    title:'Medical Writer II',
    duties:[
        {
            duty: `Wrote evidence based summaries of clinical trials aimed at serving as a resource for physicians and other health care professionals at point of care.`

        },
        {
            duty:`Wrote evidence based point of care references for a number of clinically important disease topics`
        },
        {
            duty:`Critically assessed published clinical trials for methodological integrity`
        }
    ]
  }
]